charts/hono/values.yaml

```diff
@@ -23,17 +23,39 @@ amqpMessagingNetworkExample:
   # dispatchRouter.imageName contains the name (including tag)
   # of the container image to use for the example AQMP Messaging Network Router.
   dispatchRouter:
-    imageName: quay.io/enmasse/qdrouterd-base:1.7.0
+    imageName: quay.io/interconnectedcloud/qdrouterd:1.10.0
     svc:
       annotations: {}
       loadBalancerIP:
+    # resources contains the container's 'requests and limits for CPU and memory
+    # as defined by the Kubernetes API.
+    # Refer to https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/
+    # for a description of the properties' semantics.
+    resources:
+      requests:
+        cpu: "0.1"
```

Comment:
I would suggest to go for more common format of "100m" instead of "0.1". Same for all the other locations.